solar.rb

```diff
@@ -0,0 +1,160 @@
+# our customer will be able to interact with the program and
+# choose which pieces of information they can see at a time,
+# specifically by choosing which planet.
+# sudo gem install samsouder-titlecase --source=http://gems.github.com
+
+require 'rubygems'
+require 'titlecase'
+
+class SolarSystem
+
+  attr_accessor :planet_details, :choose_planet, :planets, :planet_list, :add_planet
+
+  def initialize(planets)
+    @planets = planets
+  end
+  # has an instance method that returns the planet details based on user input
+
+  def planet_list
+    planet_array = []
+    @planets.each_with_index do |planet, index|
+      # **--->how to do without storing list in an array as a variable??
+      # **--->prints whole array of hashes if not
+      current_planet = "#{index + 1}. #{planet.name}"
+      planet_array << current_planet
+    end
+    return planet_array
+  end
+
+  def updated_list
+    updated_array = []
+    @planets.each_with_index do |planet, index|
+      current_planet = "Planet: #{index + 1}. #{planet.name}"
+      updated_array << current_planet
+    end
+    return updated_array
+  end
+
+  def add_planet
```

Comment:
This method is part of `SolarSystem` and it's mixing concerns a bit.  It's part of `SolarSystem` and also dealing with the user and keyboard input and puts.  This is a mix of roles.  Generally a class should deal with **one** concept.  In this case `SolarSystem` should focus on being a collection of planets and operations you can do on a `SolarSystem`.  So Solar System does it's thing and methods in the main program can deal with the user.  

BTW this is a very minor thing in this project